content/deep-dive/consensus-algorithm/introduction.md

```diff
@@ -6,7 +6,7 @@ menu:
         weight: 2
 ---
 
-When building a distributed system one principal goal is often to build in fault-tolerance. That is, if one particular node in a network goes down, or if there is a network partition, the systems continues to operate. The cluster of nodes taking part in a distributed consensus protocol must come to agreement regarding values, and once that decision is reached, that choice is final, even if some nodes were in a faulty state at the time.
+When building a distributed system, one principal goal is to build in fault-tolerance. That is, if one particular node in the network goes down, or if there is a network partition, the system should continue to operate in a consistent way, i.e., all nodes in the system should have a consensus on the state (or simply "values") of the system. The consensus should be considered final once it is reached, even if some nodes were in faulty states at the time of decision.
```

Comment:
I don't think "all nodes" is correct or precise given that it says "some nodes were in faulty state" later.

Reply:
Yes, agree. From a client perspective, it's the entire distributed system that should be consistent and the "consensus" of nodes in the system is kind of the actual implementation. I guess instead of "all nodes", "the system of nodes" would be a better way to refer to the whole system.

Reply:
How just "nodes"?

Reply:
Yep, that works as well. Will update.